at_client/lib/src/util/at_client_util.dart

```diff
@@ -113,26 +113,23 @@ class AtClientUtil {
   /// Accepts [AtKey] and returns [AtKey.key] with namespace appended
   /// Appends namespace if [atKey.metadata.namespaceAware] is set to true,
   /// else namespace is not appended
-  static String getKeyWithNameSpace(AtKey atKey) {
+  static String getKeyWithNameSpace(
+      AtKey atKey, AtClientPreference atClientPreference) {
```

Comment:
since are using only namespace can we replace atClientPreference with namespace?

Reply:
@murali-shris : Since the namespace in AtClientPreference is `String?`, instead of checking isNull in all the caller methods (though namespace in AtClientPreference going is most unlikely), i have passed AtClientPreferences and checking for isNull here.